.github/workflows/PROJECT-README-VERSION-UPDATE.yaml

```
@@ -71,31 +80,89 @@ jobs:
+            else
+              # 파일 맨 앞에 추가
+              sed -i '1i\<!-- 수정하지마세요 자동으로 동기화 됩니다 -->\n## 최신 버전 : '"$VERSION_TEXT"'\n' README.md
+            fi
+            echo "✅ 기본 버전 정보 추가 완료: $VERSION_TEXT"
+          else
+            # 기존 버전 라인 업데이트
+          
+            # HTML 주석이 해당 라인 위에 있는지 확인
+            LINE_NUMBER=$(grep -n -i "$PATTERN_FOUND" README.md | head -n 1 | cut -d: -f1)
+            PREV_LINE_NUMBER=$((LINE_NUMBER - 1))
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)